src/Business Foundation/App/NoSeries/src/Setup/NoSeries.Report.al

```diff
@@ -0,0 +1,197 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Foundation.NoSeries;
+
+using System.Utilities;
+
+report 21 "No. Series"
+{
+    DefaultLayout = RDLC;
+    RDLCLayout = './NoSeries/src/Setup/NoSeries.rdlc';
```

Comment:
If this is added for backwards compatibility only, then ok. Just wondering what's the added value of having a hard 'printout' / fix report to review the No series / line / relationship / line data. Aren't there any other alternatives using queries, excel layouts, ... (long term 'Analysis View')?

![image](https://github.com/microsoft/BCApps/assets/25268332/8bbdbf8c-5d61-46d9-9084-1a69989b0d24)

Reply:
Those are just old objects moved from the Base App into Business Foundation.
I'm struggling to find the right answer on whether this PR should fix also old code. But I agree for sure with everything you found for the new objects.

Reply:
I overlooked the PR description, mainly asking for input on the new **interfaces** introduced. I'm aware this is just a first run to move No.Seroes as-is functionality to the new Foundation layer + adding interface support. Hopefully in a next run, some improvements might be added as well based on the suggested feedback.

Reply:
From one side it is move from old to nrw structure but from other it is totally new module. I would love to see for it already either excel or word layput if needed. It should not be a big change even if leaving the old RDLC layout.

Reply:
We are going to obsolete these reports. For now they are still moved to Business Foundation to keep all No. Series functionality in one place.

Reply:
We have moved the reports but they are being obsoleted. As you say, this is not the new way of doing things.